modules/host/negotiatesettings.go

```diff
@@ -26,54 +26,60 @@ func (h *Host) capacity() (total, remaining uint64, err error) {
 	return total, remaining, nil
 }
 
+// externalSettings compiles and returns the external settings for the host.
+func (h *Host) externalSettings() (modules.HostExternalSettings, error) {
+	totalStorage, remainingStorage, err := h.capacity()
+	if err != nil {
+		return modules.HostExternalSettings{}, err
+	}
```

Comment:
Another approach to ditching the error is to just return an error string ("problems parsing folder") as the name if there's an error. Then you still return all of the rest of the information. That's definitely more graceful than not being able to see the status of *any* of your storage folders if only one of them is having problems.